code/modules/economy/commodity.dm

```diff
@@ -2663,6 +2663,15 @@
 	upperfluc = 500
 	lowerfluc = -500
 
+/datum/commodity/foam_dart_grenade
+	comname = "Foam Dart Grenade"
+	comtype = /obj/item/old_grenade/foam_dart
+	desc = "Goes great with foam dart guns!"
+	price = 250
+	baseprice = 250
+	upperfluc = 500
+	lowerfluc = -500
```

Comment:
The lowerfluc should be reduced so you don't have grenades priced at -250 credits, maybe set it to -50.

Reply:
okay